crates/bin/pcli/src/command/tx.rs

```diff
@@ -957,6 +957,8 @@ impl TxCmd {
                 let position = order.as_position(&asset_cache, OsRng)?;
                 tracing::info!(?position);
 
+                println!("Position id:{}", position.id().to_string());
```

Comment:
```suggestion
                println!("Position id: {}", position.id());
```
Add a space and omit redundant `to_string`